testData/solutions/win32.json5

```diff
@@ -3002,6 +3002,613 @@
+                            "description": "Unit system to be used as default",
+                            "type": "number",
+                            "enum": ["0", "1"],
+                            "outputFormat": ["Metrict", "Imperial"]
+                        }
+                    },
+                    "sNativeDigits": {
+                        "scheme": {
+                            "title": "Native digits representation",
+                            "description": "Character set for representing the native digits",
+                            "type": "string",
+                            "enum": [
+                                "0123456789",
+                                "٠١٢٣٤٥٦٧٨٩",
+                                "০১২৩৪৫৬৭৮৯",
+                                "〇一二三四五六七八九",
+                                "०१२३४५६७८९",
+                                "૦૧૨૩૪૫૬૭૮૯",
+                                "၀၁၂၃၄၅၆၇၈၉",
+                                "༠༡༢༣༤༥༦༧༨༩",
+                                "೦೧೨೩೪೫೬೭೮೯",
+                                "០១២៣៤៥៦៧៨៩",
+                                "໐໑໒໓໔໕໖໗໘໙",
+                                "൦൧൨൩൪൫൬൭൮൯",
+                                "๐๑๒๓๔๕๖๗๘๙",
+                                "౦౧౨౩౪౫౬౭౮౯",
+                                "๐๑๒๓๔๕๖๗๘๙",
+                                "۰۱۲۳۴۵۶۷۸۹"
+                            ],
+                            "outputFormat": [
+                                "Latin",
+                                "Arabic",
+                                "Bengali",
+                                "Chinese",
+                                "Devanagari",
+                                "Gujarati",
+                                "Burmese",
+                                "Gurmukhi",
+                                "Kannada",
+                                "Khmer",
+                                "Lao",
+                                "Khmer",
+                                "Tamil",
+                                "Telugu",
+                                "Thai",
+                                "Urdu"
+                            ]
+                        }
+                    },
+                    "NumShape": {
+                        "scheme": {
+                            "title": "Native digits contextual usage",
+                            "description": "Determines the context in which the native digits should be used or not",
+                            "type": "string",
+                            "enum": ["Context", "Level", "National"]
+                        }
+                    },
+                    "sCurrency": {
+                        "scheme": {
+                            "title": "System default currency",
+                            "description": "Standard currency for being used in the system",
+                            "type": "string",
+                            "enum": ["$", "€"]
```

Comment:
The £ hasn't disappeared yet.

Reply:
Hahaha sadly those ones were the default ones that appeared in my system.